### ISSUE TITLE
Inconsistent predict return value

### ISSUE THREAD

**Issue by gggdomi** [salestock/fastText.py#92]
Hi, I noticed an unexpected behavior using predict with trained vs loaded model

- Python: 3.4.3
- OSX: 10.12.1
- fasttext: 0.8.0

```python
import fasttext
model = fasttext.supervised('train_set.ft', 'my_model')
with open('test_set.ft') as f:
    data = f.read().split('\n')
    test_set = [' '.join(d.split()[1:]) for d in data if len(d) > 1]
print(model.predict(["hello"]))
# Output : [['2']]

model = fasttext.load_model('my_model.bin')
print(model.predict(["hello"]))
# Output : [['__label__2']]
```

The difference is in the default `label_prefix` value (`__label__` in `supervised` and `''` in `load_model`)

I'll send a PR with `label_prefix` as default value in both soon, so the output is `[['2']]` in both case.

However, I wonder if it would be better to output `[['__label__2']]` twice, to conform with the C++ implementation output. Any opinion on it?

By the way, it would probably be better to load the `label_prefix` value from the .bin file instead of specifying it. It's not trivial though because, if I get it right, the labels seems to be stored in the dictionary only and `label_prefix` isn't saved in the Args class.

**Comment by pyk** [comment]
@gggdomi you can load your model with specified `label_prefix` to get expected behaviour https://github.com/salestock/fastText.py#load-pre-trained-classifier

>It's not trivial though because, if I get it right, the labels seems to be stored in the dictionary only and label_prefix isn't saved in the Args class.

Yep, you are right

**Comment by gggdomi** [comment]
@pyk: indeed, but with no value specified, it is confusing to have different outputs as in my code snippet.
Is there a reason for not having `__label__` as the default value for `label_prefix` in `load_model`?

**Comment by pyk** [comment]
> Is there a reason for not having `__label__` as the default value for `label_prefix` in `load_model`?

Because `label_prefix` info is not saved in the `model.bin` file. We can't assume that the `label_prefix` is `__label__`. It should be explicitly added to prevent unexpected behaviour (We do replace string operation on the label).
